src/audio/codec/dts/dts.c

```diff
@@ -25,7 +25,7 @@ static void *dts_effect_allocate_codec_memory(void *mod_void, unsigned int lengt
 
 	comp_dbg(dev, "dts_effect_allocate_codec_memory() start");
 
-	pMem = module_allocate_memory(mod, (uint32_t)length, (uint32_t)alignment);
+	pMem = mod_alloc(mod, (uint32_t)length, (uint32_t)alignment);
```

Comment:
@jsarha to keep it super simple, can we have ```mod_alloc(mod, size)``` and ```mod_alloc_aligned(mod, size, alignment)```, this keeps it simple for module authors as in most cases they should not need to care about alignment.